packages/components/src/Inject/Inject.md

```diff
@@ -1,7 +1,94 @@
-### **INJECT**
+# **INJECT**
 
 With the Inject components you can instanciate any component anywhere. This allows a great flexibility in your component design.
 
+The concept is to let you add `slot` in your component so from the outside a user may
+inject in a controlled way some other components.
+
+# How to customize components as a user
+
+For components you can pass props like this one:
+
+```js
+    const props = {
+        components: {
+            'before-something': [
+                { component: 'Action', label: 'LabelAction1', icon: 'IconAction1' },
```

Comment:
could use some more information geared toward the component user like "component: Action" is used to describe which component you wish to inject according to the configured list of component for your app.
and adding, all other properties of the object a props that are going to be given to the Instancied component